packages/api/src/external/carequality/mock-patient-discovery.ts

```diff
@@ -35,10 +35,10 @@ export async function mockResult(id: string, cxId: string, patientId: string) {
     responseTimestamp: "2023-11-29T08:41:47.047-0800",
     patientId,
     patientMatch: true,
-    xcpdHomeCommunityId: "1.2.840.114350.1.13.11511.3.7.3.688884.100.1000",
+    xcpdHomeCommunityId: "2.16.840.1.113883.3.9621",
```

Comment:
This attribute should be called `gatewayHomeCommunityId` now. The schema was updated in [this PR](https://github.com/metriport/metriport/pull/1331), but looks like I forgot to update this mock template. Let's assign the type to this constant on initialization to enable type checking, please.

Reply:
> assign the type to this constant on initialization to enable type checking

![image](https://c.tenor.com/LjTVUzKSJfcAAAAC/tenor.gif)